scripts/docker/deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#ENV=development
+ENV=develop
+
+REGION=eu-central-1
+AWS_ACCOUNT_ID=724443507988
+#PROFILE=default
+PROFILE=personalAccount
```

Comment:
wrong ID and PROFILE

Reply:
@BohdanRSchool thanks! Didn't use this script and forgot about it. Updated